gctree/isotype.py

```diff
@@ -0,0 +1,432 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Should we expose this module in the API too? It seems like this could be useful (e.g. for transforming a tree to an isotype tree in a jupyter notebook workflow). This will probably require some tidying of public/private module members, and could replace these lines with a short module level docstring.
```suggestion
r"""A module for incorporating isotype information in gctree inference.
"""
```

Reply:
Yes, I'll do that.